.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,45 @@
+name: build-and-publish
+on: [push]
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Install opm from OpenShift Mirror
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          opm: "latest"
+
+      - name: Install skopeo
+        run: |
+          sudo apt-get -y install skopeo
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      
+      - name: Install yq
+        env:
+          VERSION: v4.14.2
+          BINARY: yq_linux_amd64
+        run: |
+          wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O /usr/local/bin/yq
+          chmod +x /usr/local/bin/yq
+      
```

Comment:
Our convention is that all CI builds should be reproducible locally as well, thus all necessary tools should installable using `make tools` target.  It may be worth understanding why some tools are needed (to reduce the maintenance effort)